libplatsupport/plat_include/bcm2837/platsupport/plat/serial.h

```diff
@@ -13,9 +13,11 @@
 
 #define UART_PADDR_0                (UART_BUSADDR-BUS_ADDR_OFFSET+PADDDR_OFFSET)
 /* Broadcom 2835 Peripheral Manual, section 7.5,
- * table "ARM Peripherals interrupts table"
+ * table "ARM Peripherals interrupts table", 
+ * 0x7E215000 is Auxiliaries, interrupt is 29,
+ * at kernal/include/drivers/bcm2836-armctrl-ic.h aux irq set to 93. 
```

Comment:
Seems the interrupt 57 was wrong all the time, because it was for the wrong UART and did not take into account the interrupt mapping. Let's rephrase this block to explain how things work and where the numbers actually come from in the RasPi. 

```suggestion
/* The mini-UART from the Auxiliary peripheral triggers interrupt 29 ("Aux int")
 * on the VideoCore, which then signals pending GPU interrupts to the ARM core. 
 * Further details can be found in the Broadcom BCM2835/BCM2837 Manual, section
 * 7.5, table "ARM peripherals interrupts table"). The seL4 kernel's interrupt 
 * controller driver creates a linear mapping of all interrupts in the system
 * (see include/drivers/bcm2836-armctrl-ic.h32) with 32 ARM PPIs, 32 basic 
 * interrupts and 64 GPU interrupts. Thus the actual interrupt to be used in 
 * userland is 93 (=32+32+29).
 */
```